connector/dialect.tdd

```diff
@@ -1054,6 +1054,11 @@
      <argument type='str' />
      <argument type='str' />
     </function>
+    <function group='string' name='ENDSWITH' return-type='bool'>
+      <formula>(CASE WHEN (CHAR_LENGTH(%1) - CHAR_LENGTH(%2)) &lt; 0 THEN false ELSE SUBSTRING(%1 FROM (CHAR_LENGTH(%1) - CHAR_LENGTH(%2)) FOR (CHAR_LENGTH(%2))) = %2 END)</formula>
```

Comment:
```suggestion
      <formula>(CASE WHEN CHAR_LENGTH(%1) &lt; CHAR_LENGTH(%2) THEN false ELSE SUBSTRING(%1 FROM (CHAR_LENGTH(%1) - CHAR_LENGTH(%2)) FOR (CHAR_LENGTH(%2))) = %2 END)</formula>
```
why not do `char_length(%1) < char_length(%2)` and save us the additional operation?

Reply:
Seems very reasonable to me 👍